libs/langchain/tests/unit_tests/schema/runnable/test_runnable.py

```diff
@@ -1063,3 +1064,42 @@ async def test_llm_with_fallbacks(
     assert await runnable.abatch(["hi", "hey", "bye"]) == ["bar"] * 3
     assert list(await runnable.ainvoke("hello")) == list("bar")
     assert dumps(runnable, pretty=True) == snapshot
+
+
+class FakeSplitIntoListParser(BaseOutputParser[List[str]]):
+    """Parse the output of an LLM call to a comma-separated list."""
+
+    @property
+    def lc_serializable(self) -> bool:
+        return True
+
+    def get_format_instructions(self) -> str:
+        return (
+            "Your response should be a list of comma separated values, "
+            "eg: `foo, bar, baz`"
+        )
+
+    def parse(self, text: str) -> List[str]:
+        """Parse the output of an LLM call."""
+        return text.strip().split(", ")
+
+
+def test_each(snapshot: SnapshotAssertion) -> None:
```

Comment:
(suggestion) there is no minimal test for testing `each`. It's great to test more complex scenarios, but I think we need to include minimal tests as well.

Here is a simple test. Double `.map` invocation is present together with what I think is the correct expected behavior

```
def test_each_simple() -> None:
    """Test that each() works with a simple runnable."""
    parser = FakeSplitIntoListParser()
    assert parser.invoke("first item, second item") == ["first item", "second item"]
    assert parser.map().invoke(["a, b", "c"]) == [["a", "b"], ["c"]]
    assert parser.map().map().invoke([["a, b", "c"], ["c, e"]]) == [
        [["a", "b"], ["c"]],
        [["c", "e"]],
    ]
```

Reply:
Added